README.md

```diff
@@ -25,6 +27,27 @@ The config passed to `Configure` can contain the following fields.
 | `snapshot`       | Whether or not the plugin will take a snapshot of the entire table before starting cdc mode, by default true.                                                                                                                                   | no       | "false"                                                |
 | `batchSize`      | Size of batch. By default is 1000. <b>Important:</b> Please don't update this variable after the pipeline starts, it will cause problem with position.                                                                                          | no       | "1000"                                                 |
 
+## Destination
+
+| name             | description                                                                                                                                                                                                                                     | required | example                                                |
+|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|--------------------------------------------------------|
+| `username`     | Snowflake username | yes      | "username" |
+| `password`     | Snowflake password | yes      | "password" |
+| `host`     | Host for snowflake instance | yes      | 443 |
+| `database`     | Database you wish to use in your Snowflake account | yes      | "userdb" |
+| `schema`     | Schema you wish to use in your Snowflake account | yes      | "STREAM_DATA" |
+| `warehouse`     | Warehouse you wish to use in your Snowflake account | yes      | "COMPUTE_WH" |
+| `stage`     | Snowflake Stage to use for uploading files before merging into destination table. | yes      | "ordersStage" |
+| `primaryKey`     | Primary key of the source data | yes      | "id" |
+| `namingPrefix`     | Prefix to append to update_at , deleted_at, create_at at destination table. Default is "meroxa_" | no      | "meroxa" (translates to `meroxa_updated_at` for update timestamps) |
+| `format`     | Data type of file we upload and copy data from to snowflake | yes      | "csv" |
+| `compression`     | Compression to use when staging files in Snowflake | no      | "gzip" |
```

Comment:
so there are 3 options `ztsd`, `gzip` and `copy` - last is no op. ztsd is at least 2x+ times faster than gzip so can we make this one the default